core/primitives/src/version.rs

```diff
@@ -84,7 +84,7 @@ pub enum ProtocolFeature {
 
 /// Current latest stable version of the protocol.
 #[cfg(not(feature = "nightly_protocol"))]
-pub const PROTOCOL_VERSION: ProtocolVersion = 41;
+pub const PROTOCOL_VERSION: ProtocolVersion = 42;
```

Comment:
why? What is the protocol change here?

Reply:
fees changed

Reply:
but I think let's going to change fees in https://github.com/near/nearcore/pull/3653 and do a one time protocol change to update fees? Since not only this PR, but other PRs have introduces many fee changes since last hard fork, and fees is not updated since then.